x-pack/test/functional/apps/lens/group6/legend_statistics.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['visualize', 'lens', 'common', 'header']);
+  const find = getService('find');
+  const listingTable = getService('listingTable');
+  const kibanaServer = getService('kibanaServer');
+
+  async function loadSavedLens(title: string) {
+    await PageObjects.visualize.gotoVisualizationLandingPage();
+    await listingTable.searchForItemWithName(title);
+    await PageObjects.lens.clickVisualizeListItemTitle(title);
+  }
+
+  async function expectLegendOneItem(name: string, value?: string) {
+    const expectedText = value ? `${name}\n${value}` : name;
+    const legendElement = await find.byCssSelector('.echLegendItem');
+    const text = await legendElement.getVisibleText();
+    expect(text).to.eql(expectedText);
+  }
+
+  async function goToTimeRangeForXyChart() {
+    await PageObjects.lens.goToTimeRange(undefined, 'Sep 22, 2015 @ 23:30:00.000');
```

Comment:
nit: to save some FTR time, can you configure this new timerange via UI settings at the beginning of the suite, then restore it base in a `after` callback?